recipes/cornac/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "cornac" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9fe801c4c72a79a48eed383ef34e9b39386f867c6feaf2e4caccaf55bb14fe7a 
+
+build:
+  skip: True  # [py<35]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - cython
+    - numpy 1.11
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy
+    - tqdm
+
+test:
+  imports:
+    - cornac
+
+about:
+  home: https://cornac.preferred.ai/
+  license: Apache 2.0 
```

Comment:
needs `license_family` and `license_file`.

You should probably add `license_file` to your `setup.cfg` so that it becomes added to the pypi distribution too
https://github.com/jupyter/qtconsole/issues/298

For now, you may copy the `LICENSE` file from your repo to this feedstock.